test/hotspot/gtest/unittest.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This copyright update isn't in the JDK 17u version of the patch, please remove.

https://github.com/openjdk/jdk17u/commit/89dd23f2fab0d98879e68f817923656e113087e3#diff-ff1dd449619fcf441d7a63b01c6c4b6bf3e6fd5dc1149ffa48f38d5e6d82e5ba

For backports we try to avoid copyright updates, only use what was done in the backports of newer code-lines.

Reply:
Ah, good to know! Updated in new commit.